src/lazy/binary/encoded_value.rs

```diff
@@ -40,7 +40,7 @@ impl EncodedHeader for Header {
 /// allowing a user to re-read (that is: parse) the body of the value as many times as necessary
 /// without re-parsing its header information each time.
 #[derive(Clone, Copy, Debug, PartialEq)]
-pub(crate) struct EncodedValue<HeaderType: EncodedHeader> {
+pub struct EncodedBinaryValue<HeaderType: EncodedHeader> {
```

Comment:
🪧 Prior to this change there was an `EncodedTextValue` and an `EncodedValue`. I renamed the latter for symmetry/clarity.